test/uk/gov/hmrc/agentclientrelationships/services/RelationshipsServiceSpec.scala

```diff
@@ -340,7 +343,7 @@ class RelationshipsServiceSpec extends UnitSpec
   }
 
   def verifyRecordNotRecreated(repository: FakeRelationshipCopyRecordRepository, record: RelationshipCopyRecord): Unit = {
-    repository.recordCreated shouldBe false
+    repository.findBy(Arn(record.arn),MtdItId(record.clientIdentifier))
```

Comment:
@cavster I don't think this method verifies that the record was not recreated any more